sources/mongo/mongo.go

```diff
@@ -0,0 +1,64 @@
+package mongo
+
+/*
+ Package mongo adapts a mgo iterator (or anything that implements
+ the interface) into a optimus.Table for consumption
+
+ example:
+   // Connect to mongo, get your collection then do the following:
+	 iter := collection.Find(nil).Limit(100).Iter()
+	 table := mongo.New(iter)
+*/
+
+import (
+	"gopkg.in/Clever/optimus.v3"
+)
+
+// Iter simulates the mgo.Iter interface so we can remain independent
```

Comment:
Maybe give the fully qualified import path of mgo? e.g. "gopkg.in/mgo.v2" in case someone doesn't know what it is (unlikely if they're using mongo and go, but still seems worthwhile)